hdf5.spec

```diff
@@ -1,29 +1,27 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 %{!?_fmoddir:%global _fmoddir %{_libdir}/gfortran/modules}
 
-%global daos_major 0
-%global hdf5_commit fa40c6c59af5d9aabd4b478cd02f8a9f7ebf7922
-%define hdf5_sha .gfa40c6c59a
+%global daos_major 1
 
 # Patch version?
 %global snaprel %{nil}
-# HDF5 versions
-%global major 1.12
-%global minor 0
+%global hdf5_major 1
+%global hdf5_minor 13_0
+%global hdf5_release rc5
 
```

Comment:
tried using --defines in Makefile so these variables are only in one place but CI would not pass.    (build #7 is an example)